BearStudio/start-ui-native
# Follow standards on the app

## Purpose
Fix #12 

## What it contains:
 - Remove the config files managed with react-native-dotenv and replaced them with subfolders in the `environment` folder. The documentation has been updated too
 - Edited the build scripts, package.json to have script to change the environment files
 - Added an "About" page containing the app's version number, dev / prod mode, and so on.
 - Splitted the big App.js into 3 files (App.js, Providers.js, Routing.js)
 - Added SafeArea

## How to test
 - Run the app
 - Build the app (Android and iOS)
 - Check that everything is still working
 - Check the About page and ensure that everything's right
